person.rb

```diff
@@ -1,12 +1,14 @@
-class Person
+require_relative 'nameable'
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
 
   def initialize(age, name = 'Unknown', parent_permission: true)
+    super
```

Comment:
- [ ] Please consider adding the parenthesis when you call the `Nameable` constructor, this is because if you call it without the parenthesis, by default it will pass the same arguments as the child constructor `age`, `name`, `parent_permission` but your `Nameable` doesn't expect any arguments, ending up with an `Argument Error`.

```suggestion
    super()
```

Please consider adding them! 🤓 🔧